example/lambdas/ftpPopulateTestLambda/spec

```diff
@@ -0,0 +1 @@
+../../spec
```

Comment:
Note: this is how git represents a symlink.   This is included to make use of a spec helper